src/plugins/intel_npu/tools/multi-provider-inference-tool/params.py

```diff
@@ -0,0 +1,657 @@
+                data["convert"]["shape"] = shape_to_list(data["convert"]["shape"])
+            if "layout" in data["convert"].keys():
+                data["convert"]["layout"] = layout_to_str(data["convert"]["layout"])
+
+    def __validate_bin__(data):
+        if "shape" in data.keys():
+            data["shape"] = shape_to_list(data["shape"])
+        if "layout" in data.keys():
+            data["layout"] = layout_to_str(data["layout"])
+
+    def __validate__(files_per_input_json) :
+        for i,d in files_per_input_json.items():
+            if "type" not in d.keys():
+                raise RuntimeError(f"FilesStorage validity failed: important field \"type\" is absent for input source: \"{i}\": {d}\n. Please specify a one from: {FilesStorage.source_file_types}")
+            if d["type"] not in FilesStorage.source_file_types:
+                raise RuntimeError(f"FilesStorage validity failed: unsupported \"type\" value: \"{d['type']}\" for input source: \"{i}\": {d}\n. Please specify a correct one from: {FilesStorage.source_file_types}")
+            if not FilesStorage.source_json_schema_allowable[d["type"]].issubset(set(d.keys())):
+                raise RuntimeError(f"FilesStorage validity failed: some important fields: {FilesStorage.source_json_schema_allowable[d['type']]} are absent in the description of input source: \"{i}\": {d}\n. Please specify them all")
+            if len(FilesStorage.source_json_schema_forbidden[d["type"]] & (set(d.keys()))):
+                raise RuntimeError(f"FilesStorage validity failed: These fields: {FilesStorage.source_json_schema_forbidden[d['type']]} are not allowed in the description of input source: \"{i}\": {d}\n. Please remove them all.")
+
+            if d["type"] == FilesStorage.source_file_types[int(FilesStorage.SourceType.image)]:
+                FilesStorage.__validate_image__(d)
+            if d["type"] == FilesStorage.source_file_types[int(FilesStorage.SourceType.bin)]:
+                FilesStorage.__validate_bin__(d)
+
+    @staticmethod
+    def __parse_json_obj__(json_data):
+        return_json_data = {}
+        for io_name, io_data in json_data.items():
+            if isinstance(io_data, Mapping):
+                return_json_data[io_name] = io_data
+                if "type" not in io_data.keys():
+                    return_json_data[io_name]["type"] = FilesStorage.SourceType.bin.name
+            else:
+                return_json_data[io_name] = {"files": io_data, "type" :  FilesStorage.SourceType.image.name}
+        FilesStorage.__validate__(return_json_data)
+        return return_json_data
+
+    def parse_inputs(self, console_input_files_list_per_model: str):
+        if if_file(console_input_files_list_per_model):
+            with open(console_input_files_list_per_model) as file:
+                json_input = json.load(file)
+        else:
+            try:
+                json_input = json.loads(console_input_files_list_per_model)
```

Comment:
Example string of using JSON from cmd to configure inputs 
```
 "{\"result.1\":{\"files\":[\"C:\\miniImageNet\\1.JPEG\"], \"type\":\"image\"}}"
```

looks cumbersome. Unfortunately, the standard JSON parse won't consume `'` as string-decorator rather than `"`   which must be shield by `\` when it appear as a part of a command-line argument. Perhaps it makes sense to insert json-cmd-arg-preprocessor, which can transform more short-notations like  `"{'result.1':{'files':['C:\miniImageNet\1.JPEG'], 'type':'image'}}"` into a valid JSON string suitable for json loading